RELEASE-NOTES.txt

```diff
@@ -2,6 +2,7 @@
 
 18.7
 -----
+* [**] Posts List: Adds copy post link functionality [https://github.com/wordpress-mobile/WordPress-Android/pull/15570]
```

Comment:
Thank you for thinking of our users and adding a release note on this @ovitrif 👍 
Adding this comment as a reminder to move the release not to the next release since `18.7` is cut today.

Reply:
Thank you for the remark! I noticed @AliSoftware usually updates the release notes with the new sections, would that mean it's better not to do it myself or I can proceed to add the new section?

P.S. Also asking because of a rebase issue which forces me to push a new change to release notes 🙃

Reply:
We update the release notes section right after we do the Code Freeze on start of each sprint then merge the new `release/*` branch we just cut back into `develop` for branch synchronization.

So indeed you usually don't have to create those new section titles in theory.

In practice, when you created your PR before the code freeze, and then the code freeze happened and added the new section for the new version after your branch was cut, I recommend you merge `develop` back into your PR branch at that point to:
 - get the new section title in `RELEASE-NOTES.txt`… allowing you to either then add the entry for your PR under it, or  move the entry you previously added to the `RELEASE-NOTES.txt` file (if you already did did so in preparation for the previous release that your PR missed) under the new one instead;
 - get any new stuff from `develop` that might have moved on since (thus reducing the risk of potential conflicts between your branch and develop)

_In contrast, trying to add the new section yourself in your PR will just likely generate a git conflict, with 2 identical new `18.8` lines not completely in sync potentially confusing git… thus not the best option_

Reply:
For reference, my PR adding this new section title (as part of the code freeze sync) is [here](https://github.com/wordpress-mobile/WordPress-Android/pull/15574/files#diff-558ce8820a544da19f7fccdb2d1d661e46d68c130a1a39abdd222aa3484a5873R3) (and still pending review and not yet landed in `develop` as I write those lines)

Reply:
Many thanks @AliSoftware for the detailed insights into this procedure 🙇🏻‍♂️

Reply:
Seeing that the pull-request was merged I updated the change to release-notes

Reply:
Thank you for the change @ovitrif 🙇